### ISSUE TITLE
Potential normalization issues for conversion between CartesianTensor and IrrepTensor

### ISSUE THREAD

**Issue by blondegeek** [e3nn/e3nn#119]
If I want to represent an ellipsoid, I can write down a 3x3 Cartesian identity matrix with and scale the matrix along the x, y, z axes accordingly. If I convert this 3x3 Cartesian matrix to an `IrrepTensor`, I "should" be able to recover the desired ellipse when I plot, however, this is not what occurs (typically the signal has dents meaning the scalar component is underweighted). This is likely because of differences of normalizations between `CartesianTensors` and `IrrepTensors`. This is likely not an issue for any training tasks, however it might be nice to figure out for ease of visualization because there will be a difference between the coefficients of projecting an ellipsoid onto our spherical harmonics vs. taking a CartesianTensor representation and converting to an IrrepTensor.

**Comment by mariogeiger** [comment]
I am afraid that there exist no universal solution to this problem. Each application will need its own weighting. 
We can look for weights that makes sense and add a `normalization` argument to the conversion function

**Comment by mariogeiger** [comment]
Added a `normalization` argument in `SphericalTensor.plotly_surface`
```
s = SphericalTensor.from_irrep_tensor(x.to_irrep_tensor())
surface = go.Surface(**s.plotly_surface(normalization=torch.tensor([1, 1.0, 10**-0.5])))
fig = go.Figure(data=[surface])
fig.show()
```
ffff62d64232c4ee29ab933af2c5e94cd7337c45
